includes/class-hooks.php

```diff
@@ -382,6 +388,70 @@ public static function disable_masterbar( $modules, $min_version, $max_version )
 
 		return $modules;
 	}
+
+	/**
+	 * Replace default URL with current site URL.
+	 * Replace placeholder content with data from settings.
+	 *
+	 * @param WP_Post $post Post to import.
+	 *
+	 * @return WP_Post
+	 */
+	public static function wp_import_post_data_raw( $post ) {
+		$search_terms = [
+			'https://elections.usdr.dev',
+			'https:\/\/elections.usdr.dev', // Blocks use escaped versions of the URL.
+		];
+
+		$replace_terms = [
+			home_url(),
+			str_replace( '/', '\/', home_url() ),
+		];
+
+		$county_name            = get_option( 'ctcl_county_name' );
+		$state_name             = get_option( 'ctcl_state' );
+		$city_name              = get_option( 'ctcl_city' );
+		$elected_official_name  = get_option( 'ctcl_official_name' );
+		$elected_official_title = get_option( 'ctcl_official_title' );
+
+		$search_to_replace_map = [
+			'[Insert: Address]'                => get_option( 'ctcl_address' ),
+			'[Insert: Address line 2]'         => get_option( 'ctcl_address2' ),
```

Comment:
Add some logic (after the first replace call) to replace `<br>[Insert: Address line 2]` with `''`